tests/integration/components/sl-textarea-test.js

```diff
@@ -0,0 +1,305 @@
+import Ember from 'ember';
+import { moduleForComponent, test } from 'ember-qunit';
+import { skip } from 'qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent( 'sl-textarea', 'Integration | Component | sl textarea', {
+    integration: true
+});
+
+test( 'Default classes are applied', function( assert ) {
+    this.render( hbs`
+        {{#sl-textarea}}
+        {{/sl-textarea}}
+    ` );
+
+    assert.equal(
+        this.$( '>:first-child' ).prop( 'class' ),
```

Comment:
`this.$( '>:first-child' ).prop( 'class' )` returns a string. This test is a false positive, if you change `assert.equal` to `assert.strictEqual` you will see that this test fails.